### PR TITLE
fix(mail): rm brackets[] from plain text anchors, add space after urls

### PR DESCRIPTION
**Pull request by tpreusse**
```diff
Haben Sie einen Zugriff auf republik.ch angefordert?
-Dann klicken Sie bitte auf den folgenden Link: 
+Dann klicken Sie bitte auf den folgenden Link. 

-Zugriff bestätigen [{{LOGIN_LINK}}]
+Zugriff bestätigen: 
+{{LOGIN_LINK}} 

-Falls Sie diese Zugriffsanfrage nicht ausgelöst haben, können Sie die
-Zugriffsaufforderung ablehnen:
-
-[{{LOGIN_LINK}}&noAutoAuthorize=true]
+Falls Sie diese Zugriffsanfrage nicht ausgelöst haben, 
+können Sie die Zugriffsaufforderung ablehnen: 
+{{LOGIN_LINK}}&noAutoAuthorize=true 

```

### CODE REVIEW

File: packages/mail/script/generateTextTemplates.js [hunk 1]
```diff
@@ -30,13 +31,59 @@ const run = async () => {
...
+                }
+
+                const storedCharCount = options.lineCharCount
+                const text = getText()
+                const href = getHref()
+                function normalizeUrl (url = '') {
+                  return url
+                    .replace(/\s+/g, '')
+                    .replace(/^(https?:\/\/)?(www\.)?/, '')
+                    .replace('{{frontend_base_url}}', 'republik.ch')
```
Comment (patte) [inline packages/mail/script/generateTextTemplates.js:62]: why is this needed?

File: packages/mail/script/generateTextTemplates.js [hunk 2]
```diff
@@ -30,13 +31,59 @@ const run = async () => {
       await fs.writeFile(
         file.replace(/\.html$/gi, '.txt'),
         htmlToText
-          .fromString(html, { ignoreImage: true })
-          .replace(/^ /gm, '')
-          .replace(/kontakt@republik\.ch \[kontakt@republik\.ch\]/gm, 'kontakt@republik.ch'),
```
Comment (patte) [inline packages/mail/script/generateTextTemplates.js:35]: this is not needed anymore?

File: packages/mail/script/generateTextTemplates.js (hunk 1 above)
Comment (tpreusse) [reply packages/mail/script/generateTextTemplates.js:62]: to avoid `www.republik.ch {{frontend_base_url}} ` which would result in `www.republik.ch https://www.republik.ch/ `

Now it will just generate `{{frontend_base_url}}` which results in `https://www.republik.ch/ ` (hopefully clickable)

File: packages/mail/script/generateTextTemplates.js (hunk 2 above)
Comment (tpreusse) [reply packages/mail/script/generateTextTemplates.js:35]: it's handled by `toHideSameLink` (aka `hideLinkHrefIfSameAsText` option which is baked into our custom format anchor)

Comment (patte) [reply packages/mail/script/generateTextTemplates.js:35]: yes, yes, after understanding the first question, I now also see this.
